R/ggplot-global.R

```diff
@@ -44,3 +44,5 @@ ggplot_global$all_aesthetics <- .all_aesthetics
 )
 
 ggplot_global$base_to_ggplot <- .base_to_ggplot
+
+grid_has_multipath <- packageVersion('grid') >= "3.6"
```

Comment:
I don't think computing this value at build time is correct. Instead, I think you should just inline into `geom_polygon()`.